doc/nrf/app_dev/device_guides/pmic/npm2100.rst

```diff
@@ -0,0 +1,132 @@
+.. _ug_npm2100_developing:
+.. _ug_npm2100_gs:
+
+Developing with the nPM2100 PMIC
+################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| provides support for development with the `nPM2100 Power Management IC (PMIC) <nPM2100 product website_>`_, using the `nPM2100 Evaluation Kit (PCA10152) <nPM2100 EK product page_>`_.
+
+.. _ug_npm2100_features:
+
+nPM2100 features
+****************
+
+nPM2100 is a PMIC designed for primary (non-rechargeable) batteries in an extremely compact form factor.
+It has an ultra-efficient boost regulator, a dual purpose LDO/load switch, two GPIOs, an ADC, and other features.
+
+nPM2100 is the perfect companion for nRF52, nRF53, and nRF54 Series SoCs.
+It is ideal for compact and advanced IoT products that uses primary cell batteries.
+
+For additional information on the nPM2100 PMIC and associated kits, see the `nPM2100`_ technical documentation, especially the `nPM2100 EK User Guide`_.
+
+.. _ug_npm2100_compatible_boards:
+
+Boards compatible with nPM2100
+******************************
+
+The following boards in the `Zephyr`_ open source project and in the |NCS| are compatible with nPM2100 PMIC through the nPM2100 EK:
+
+OPEN QUESTION: Should we only list boards that we have overlays for in the samples, even though all nRF5 SoCs would support the PMIC if given an overlay?
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK or Prototype platform
+     - Companion module
+     - PCA number
+     - Board target
+     - Documentation
+   * - :ref:`zephyr:nrf54h20dk_nrf54h20`
+     - nPM2100 EK
+     - PCA10175
+     - | ``nrf54h20dk_nrf54h20_cpuapp``
+       | ``nrf54h20dk_nrf54h20_cpurad``
+       | ``nrf54h20dk_nrf54h20_cpuppr``
+     - | `Objective Product Specification <nRF54H20 Objective Product Specification 0.3.1_>`_
+       | :ref:`Getting started <ug_nrf54h20_gs>`
+   * - :ref:`zephyr:nrf54l15dk_nrf54l15`
+     - nPM2100 EK
+     - PCA10156
+     - | ``nrf54l15dk/nrf54l15/cpuapp``
+     - --
+   * - :ref:`zephyr:nrf5340dk_nrf5340`
+     - nPM2100 EK
+     - PCA10095
+     - ``nrf5340dk/nrf5340/cpuapp``
+     - | `Product Specification <nRF5340 Product Specification_>`_
+       | `Quick Start app`_
+       | `User Guide <nRF5340 DK User Guide_>`_
+   * - :ref:`zephyr:nrf52840dk_nrf52840`
+     - nPM2100 EK
+     - PCA10056
+     - ``nrf52840dk/nrf52840``
+     - | `Product Specification <nRF52840 Product Specification_>`_
+       | `Quick Start app`_
+       | `User Guide <nRF52840 DK User Guide_>`_
+   * - :ref:`zephyr:nrf52dk_nrf52832`
+     - nPM2100 EK
+     - PCA10040
+     - ``nrf52dk/nrf52832``
+     - | `Product Specification <nRF52832 Product Specification_>`_
+       | `Quick Start app`_
+       | `User Guide <nRF52 DK User Guide_>`_
+
+PMIC samples and libraries
+**************************
+
+The |NCS| provides several :ref:`pmic_samples` that demonstrate the features and capabilities of nPM2100 using the nPM2100 EK.
```

Comment:
"Several" is a bit much to say as we only have two samples (Fuel gauge and One button). The same goes for the corresponding doc for nPM 1300.

Reply:
True, but we would like to add more samples